packages/services/api/src/modules/auth/lib/authz.spec.ts

```diff
@@ -213,4 +213,32 @@ describe('Session.assertPerformAction', () => {
       .catch(err => err);
     expect(result2).toBeInstanceOf(AccessError);
   });
+
+  test('Allow if no resource was specified on the permission', async () => {
+    const orgId = 'aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa';
+    const bId = 'bbbbbbbb-bbbb-bbbb-bbbb-bbbbbbbbbbbb';
+    const cId = 'cccccccc-cccc-cccc-cccc-cccccccccccc';
+
+    const session = new TestSession([
+      {
+        effect: 'allow',
+        action: 'appDeployment:create',
+        resource: [],
+      },
+    ]);
```

Comment:
If you want to allow creation on app deployments on any target the resource must be

```
{
  effect: 'allow',
  action: 'appDeployment:create',
  resource: [`hrn:aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa:organization/*`],
},
```

or

```
{
  effect: 'allow',
  action: 'appDeployment:create',
  resource: [`hrn:aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa:target/${cId}`],
},
```

or

```
{
  effect: 'allow',
  action: 'appDeployment:create',
  resource: [`hrn:aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa:project/${bId}`],
},
```